tests/Test/AbstractFixerTestCase.php

```diff
@@ -494,7 +500,11 @@ private function getLinter(): LinterInterface
     private static function assertValidDescription(string $fixerName, string $descriptionType, string $description): void
     {
         self::assertMatchesRegularExpression('/^[A-Z`].+\.$/s', $description, sprintf('[%s] The %s must start with capital letter or a ` and end with dot.', $fixerName, $descriptionType));
-        self::assertStringNotContainsString('phpdocs', $description, sprintf('[%s] `PHPDoc` must not be in the plural in %s.', $fixerName, $descriptionType));
+        self::assertSame(
+            0,
+            substr_count($description, 'phpdocs') - substr_count($description, '`phpdocs_'), // allow the option to have "`phpdocs_*`"
+            sprintf('[%s] `PHPDoc` must not be in the plural in %s.', $fixerName, $descriptionType),
+        );
```

Comment:
I think right now we have _only_ `phpdocs_` (plural) as rule option, is it the case?

with that, maybe we should keep the opposite rule here, to have it plural for option?

or likely no - as we have only single rule having option like that, maybe we do not have enough options to decide if there should be a rule for it, sounds too early for me.


Original idea was to keep proper casing, which makes sense to have always the same casing for same, hm..., proper name. (polish "nazwa wlasna")

now , you suggest to unify using plural or singular, which really depends on the context and maybe should not be unified?

Reply:
I'd say they should be unified, but let's start with not allowing more plural usages.

Reply:
only if whole option make sense when using singular form. not sure if that will be the case or not as we do not have many cases yet to validate that.